### PR TITLE
10178 move all incubator modules of elico8 to public GitHub elico account extra

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

File: payment_utf8 /README.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================
+Elico Account Extra
```
Comment (duanyp1991) [inline payment_utf8 /README.rst]: remove Elico

**Comment by Rona111** [comment]
cc @elicoidal  @seb-elico @duanyp1991 @siyuancn

File: payment_utf8 /__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+# © 2015 Elico corp (www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'payment_utf8',
+    'version': '8.0.1.0.1',
+    'author': 'Elico Corp',
+    'website': 'http://www.elico-corp.com',
+    'depends': [
+        'base',
+        'account',
```
Comment (siyuancn) [inline payment_utf8 /__openerp__.py]: no need depends on base (account is already depends from base)

File: payment_utf8 /README.rst (hunk 1 above)
Comment (elicoidal) [inline payment_utf8 /README.rst]: @duanyp1991 Actually I am confused by this module: it states that it is to fix UTF8 but it seems to link paypal and accounting payments.
Module name and description should be fixed accordingly: can you help with that?

File: payment_utf8 /__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico corp (www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'payment_utf8',
```
Comment (elicoidal) [inline payment_utf8 /__openerp__.py]: Check my previous remark

File: payment_utf8 /__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    'version': '8.0.1.0.1',
+    'author': 'Elico Corp',
+    'website': 'http://www.elico-corp.com',
+    'depends': [
+        'base',
+        'account',
+    ],
+    'installable': True,
+    'application': False,
```
Comment (elicoidal) [inline payment_utf8 /__openerp__.py]: not necesary

File: payment_utf8 /invoice.py [hunk 5]
```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico corp (www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp.osv import osv, fields
```
Comment (elicoidal) [inline payment_utf8 /invoice.py]: alphabetical order

**Comment by Rona111** [comment]
cc @elicoidal  @duanyp1991 @seb-elico  @siyuancn

**Comment by elicoidal** [comment]
LGTM

**Comment by siyuancn** [comment]
LGTM

**Pull request merged**